### PR TITLE
Add try what you'll build.

### PR DESCRIPTION
**Pull request by jj-zhang**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -78,6 +78,43 @@ https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service^].
...
+```
+mvn liberty:run
+```
+
+After you see the following message, your application server is ready.
+
+[role="no_copy"]
+----
+Press the Enter key to run tests on demand.
+
```
Comment (NimG98) [inline README.adoc]: Need to end with `----` here

**Comment by jj-zhang** [comment]
Update has been made, ready for review.

File: README.adoc [hunk 2]
```diff
@@ -78,6 +78,47 @@ https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service^].
 include::{common-includes}/gitclone.adoc[]
 
 
+=== Try what you'll build
+
+The `finish` directory in the root of this guide contains the finished application. Give it a try before you proceed.
+
+To try out the application, first go to the `finish` directory and run the following
+Maven goal to build the application and deploy it to Open Liberty:
```
Comment (NimG98) [inline README.adoc:86]: The description of `mvn generate-resources` is actually to build your Angular front-end, 
not "build the application and deploy it to Open Liberty"

File: README.adoc [hunk 3]
```diff
@@ -78,6 +78,47 @@ https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service^].
...
+[role='command']
+```
+mvn liberty:run
+```
+
+After you see the following message, your application server is ready.
+
+[role="no_copy"]
+----
+Press the Enter key to run tests on demand.
```
Comment (NimG98) [inline README.adoc:104]: The output for `mvn liberty:run` is different.

Should be
> The defaultServer server is ready to run a smarter planet.

File: README.adoc (hunk 2 above)
Comment (NimG98) [reply README.adoc:86]: The "run the following Maven goal to build the application and deploy it to Open Liberty:" part should be the description for the `mvn liberty:run` command

File: README.adoc [hunk 4]
```diff
@@ -78,6 +78,47 @@ https://openliberty.io/guides/rest-intro.html[Creating a RESTful web service^].
...
+----
+
+Then, point your browser to the web application root http://localhost:9080[http://localhost:9080^] to see the following output:
```
Comment (NimG98) [inline README.adoc:107]: the url should be `http://localhost:9080/app`

**Comment by NimG98** [comment]
- [ ] copyright year add 2020

**Comment by NimG98** [comment]
copyright year should add 2020 to the original, meaning be `2019, 2020`

**Pull request merged**